### ISSUE TITLE
MCDU loading wrong runway STAR

### ISSUE THREAD

**Issue by DFozzle** [flybywiresim/a32nx#4914]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-05-26T09:08:11+00:00",
  "ref": "master",
  "sha": "a4c7ec6882f1b36cb65435f0c4ceb0e423ed6a09",
  "actor": "tracernz",
  "event_name": "manual"
}
```

### Describe the bug

Entering YSSY-YBBN and loading the BLAKA 4A 19L STAR - It loads the waypoints which align with 01R instead of 19L. 

### Expected behavior

Expected STAR to contain BLAKA DOGMU LAGEV PUBIV ISPON OMGUV BETSO 19L

### Steps to reproduce

1. Plan YSSY-YBBN in MCDU
2. Set up ARRIVAL as 19L/BLAKA 4A Arrival (VIA BETSO or NO VIAS, either way)
3. Check legs on F-PLN. Found that BLAKA VELKA BERTI PONOD ENTOX VATRO PULEX GLENN was there instead of the required BLAKA DOGMU LAGEV PUBIV ISPON OMGUV BETSO

### References (optional)

[BBNSR46-164_25MAR2021.pdf](https://github.com/flybywiresim/a32nx/files/6548456/BBNSR46-164_25MAR2021.pdf)

### Discord Username (optional)

DFozzle#1810

**Comment by tracernz** [comment]
MSFS default nav data, or Navigraph nav data?

**Comment by DFozzle** [comment]
@tracernz I've tried both. Having a look around the STARs load fine on other aircraft (MSFS Default nav data now). I did it on the CJ4 (WT Mod) and TBM930 (WT Mod) and a number of other aircraft and it seems to be limited to the A32NX.

**Comment by ManyPandas** [comment]
I've had a similar issue with STARs into KATL. I select an arrival runway and approach, then specify the STAR and transition. Especially with KATL (but with other airports too) the entirely wrong STAR procedure is loaded for the runway. When the STAR chart calls for waypoints north of the airport, the MCDU displays waypoints on the south side of the airport for an entirely different runway. It still creates a straight path to the initial approach fix, but is still nonetheless wrong. 

Not only is the STAR wrong in these cases, it also tends to truncate several waypoints, resulting in a strange leg leading directly to the initial approach fix, skipping several of the arrival waypoints.

**Comment by 2hwk** [comment]
Fixed by #4953 closing ☑️
